package.json

```diff
@@ -67,24 +67,24 @@
     "babylon": "^6.18.0",
     "chai": "^4.2.0",
     "coveralls": "^3.0.6",
-    "cross-env": "^4.0.0",
+    "cross-env": "^7.0.2",
```

Comment:
cross-eng v7 only supports node 10+, so we can't use it.

Reply:
It's maybe the time to drop old engines? I mean at least node <= 8. I pretty sure that if people does not update node, they are not updating eslint-plugin-import either.

Reply:
Nope, as far as I'm concerned it'll never be the time. Plenty of people update their deps while stuck on older nodes.

Reply:
You mean that they update `eslint-plugin-import` but not `eslint` (which now require node >= 10.12)? I seems really unlikely to me. :/

I will close this issue then, you don't seem to be ready to update before some decades.

Reply:
yep, that's indeed likely; i've worked in codebases that have done it.

Reply:
Can I ask the reason to have this strange codebase update policy? I am curious :) I don't see any reason to not being able to update node, there is almost only non breaking changes between versions, almost only additions and bug/security fixes.

Reply:
Maximum back compact helps the ecosystem. Dropping support for old platforms makes it harder for those users to upgrade - and updating node is *very* difficult when you have a sizable app, for many reasons (binary deps, performance concerns, organizational fear)

Why would we need to update any of these deps? mocha and nyc and semver, for example, work perfectly for us. Why fix what isn’t broken?

Reply:
Not broken? It is for potential contributors like me because of what I said in description if you have read it and also in your own [AppVeyor results](https://ci.appveyor.com/project/benmosher/eslint-plugin-import/builds/32988451). 
What the point of putting this CI in place if you don't care of the result?

nyc seems to be the reason of the error so I updated it and I updated some others which seems good to me if the CI results are good too.

My only objective with this pull request was to be able to help for other issues but as it is today it is impossible (probably because I am on Windows and I can't use WSL sadly). 

Just to explain my opinion on this, updating libraries is not just a fancy thing, it helps to obtain help when searching on documentation/issues/etc when an error happens and also in development dependencies. From my experience, updated libraries have always been one of the best indicator of an application/library health.

Thank you for your response :)

Reply:
Thanks, that is a fair point - although these versions used to work on appveyor, so something else changed, and presumably we could fix whatever that is without changing deps.

I very much appreciate that a non-WSL Windows user is in a tough spot.

If mocha 3 isn't viable moving forward, then I'd prefer to switch the entire test suite to the only test framework that maintains back compat, tape - but I'm not sure that would address nyc's issue.

If there's a way to get Windows tests passing without updating anything, I would be quite happy about it :-)